frontend/src/components/layout/Navbar.vue

```diff
@@ -25,14 +25,18 @@ const { getIsAuthenticated } = storeToRefs(useAuthStore());
             v-if="getIsAuthenticated"
             class="mr-2"
           >
-            <router-link :to="{ name: RouteNames.LIST_BUCKETS }">
+            <router-link 
+              :to="{ name: RouteNames.LIST_BUCKETS }"
+              aria-label="My Buckets"
+            >
               My Buckets
             </router-link>
           </li>
           <li class="mr-2">
             <a
               target="_blank"
               href="https://github.com/bcgov/bcbox/wiki"
+              aria-label="BCBox Wiki"
```

Comment:
Maybe this should be 'BCBox Help' 🤷 but either is fine by me

Reply:
I agree, we refer to it as "Help" in the interface so "BCBox Help" gets my vote